stub/pyproject.toml

```
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "asbestos-snow"
+version = "?????"  # this will be replaced in the CI pipeline
```

I would do something more explicit like...


```suggestion
version = "{CI_PIPELINE_VERSION}"  # this will be replaced in the CI pipeline
```

...but it doesn't matter.